schemas/v3.1/schema.yaml

```diff
@@ -0,0 +1,836 @@
+              - $ref: '#CommonFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          password:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+
+          clientCredentials:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+              
+          authorizationCode:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+        unevaluatedProperties: false
+
+        $defs:
+          Common:
+            $id: '#CommonFlow'
+            properties:
+              refreshUrl:
+                type: string
+                format: uri-reference
+              scopes:
+                type: object
+                additionalProperties:
+                  type: string
+          Token:
+            $id: '#TokenFlow'
+            required:
+              - tokenUrl
+            properties:
+              tokenUrl:
+                type: string
+                format: uri-reference
+          Authorization:
+            $id: '#AuthorizationFlow'
+            required:
+              - authorizationUrl
+            properties:
+              authorizationUrl:
+                type: string
+                format: uri-reference
+
+      Link:
+        $id: '#Link'
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - properties:
+                  operationRef:
+                    type: string
+                    format: uri-reference
+                  operationId: false
+              - properties:
+                  operationRef: false
+                  operationId:
+                    type: string
+            properties:
+              parameters:
+                type: object
+                additionalProperties: true
+              requestBody: true
+              server:
+                $ref: '#Server'
+            unevaluatedProperties: false
+
+      Callback:
+        $id: '#Callback'
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+
+      Schema:
+        $id: '#Schema'
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: 'http://json-schema.org/draft-04/schema#'
```

Comment:
draft/2020-12?

Reply:
actually, I think this should be `https://spec.openapis.org/oas/3.1/dialect/base`?

Reply:
I'm leaving that for when #2467 is complete, as I'm defining the schema extensions below.